util/test/sub_test

```diff
@@ -787,14 +787,12 @@ else:
 #   or not this is a good idea, but preserving it for now for backwards
 #   compatibility.
 #
+
+directoryCompopts = list(' ')
```

Comment:
The old default was `[]`, which is different than `list(' ')`. Was that intentional? The former is False-y and the latter is True-thy.